raven/utils/serializer/base.py

```diff
@@ -174,7 +174,8 @@ class FloatSerializer(Serializer):
     types = (float,)
 
     def serialize(self, value, **kwargs):
-        return float(value)
+        # Wrap with repr to convert inf/nan to string
+        return repr(float(value))
```

Comment:
Should we also normalize Ints/Longs/etc?

Reply:
I was trying to keep this patch as minimal as possible but we can do that too